CONTRIBUTING.rst

```diff
@@ -188,7 +181,7 @@ Guidelines
 * All code should be documented, to the same
   `standard <https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard>`_ as NumPy and SciPy.
 * For new functionality, always add an example to the gallery (see
-  :ref:`Sphinx-Gallery<sphinx_gallery>` below for more details).
+  :ref:`Gallery` below for more details).
```

Comment:
For some curious reason this jumps to https://matplotlib.org/stable/gallery/index.html#gallery (see here https://output.circle-artifacts.com/output/job/a2d85fcd-651a-4091-9600-277c08f7d8f3/artifacts/0/doc/build/html/development/contribute.html#guidelines) rather then our own section. Probably intersphinx but no idea why matplotlib's target takes precedence. 

Do we need to define our own unique anchor here?